### PR TITLE
Add capi visualizer

### PR DESCRIPTION
**Pull request by sbueringer**

### CODE REVIEW

File: your-first-cluster.md [hunk 1]
```diff
@@ -101,9 +106,9 @@ spec:
     serviceDomain: "cluster.local"
   topology:
     class: quick-start                   # The name of the ClusterClass the Cluster is templated from.
+    version: v1.24.6                     # The version of the Kubernetes Cluster.
```
Comment (sbueringer) [inline your-first-cluster.md:109]: Just moved it up as it seemed more "high-level" / important than the things below

File: your-first-cluster.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ A Cluster API Management Cluster is a Kubernetes Cluster where the components th
 The first step in creating a management cluster is to create a Kubernetes cluster to host the CAPI components.
 
 ```bash
+cd kubecon-na-22-capi-lab
```
Comment (sbueringer) [inline your-first-cluster.md:31]: Just as a hint that they have to be in this folder going forward

File: your-first-cluster.md [hunk 3]
```diff
@@ -44,6 +46,9 @@ In this case we're using the Docker infrastructure provider - so we need a the D
 Cluster API's CLI - `clusterctl` is used to install the CAPI Management Cluster components - luckily it's able to handle installing all of the above as well as the Custom Resource Definitions used by the Cluster API controllers.
 
 ```bash
+# Please ensure CLUSTERCTL_REPOSITORY_PATH is set and points to the clusterctl repository. 
+export CLUSTERCTL_REPOSITORY_PATH=$(pwd)/clusterctl/repository
```
Comment (sbueringer) [inline your-first-cluster.md:50]: Just as a reminder. I'm not 100% sure folks necessarily have this still set in the current shell from prereqs (especially if they did prereqs before)

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Comment by sbueringer** [comment]
Thx!

**Pull request merged**